### ISSUE TITLE
Directory with hidden files doesn't collapse

### ISSUE THREAD

**Issue by siriux** [atom/tree-view#670]
I have some directories with multiple files and subdirectories, where anything but one directory is hidden (by IgnoredNames). 

In this case I would expect the directory to collapse on the visible subdirectory, but it doesn't.

Cheers

**Comment by keskiju** [comment]
If by collapse you mean squashing of directory names, I noticed the same problem. I'm currently working with an old Java project that still uses CVS. The package folders do not squash because each of them contain a CVS folder. I have configured CVS folders to be hidden in tree view and the squash directory names feature should respect that setting. I'm using Atom 1.3.0 and my configs are:

```
  core:
    ignoredNames: [
      "CVS"
      ...
    ]

  "tree-view":
    hideIgnoredNames: true
    hideVcsIgnoredFiles: true
    squashDirectoryNames: true
```

**Comment by siriux** [comment]
I mean exactly  that, but the hidden directories are not CVS but some directories needed by my build system.

Good to know there is interest on this.

**Comment by textbook** [comment]
I am having exactly this problem with some Java files - it was working fine until I used the Finder to navigate into the collapsed directories, adding a `.DS_Store` and forcing them to un-collapse in Atom:

```
|____groovy
| |____.DS_Store
| |____tld
| | |____.DS_Store
| | |____org
| | | |____.DS_Store
| | | |____package
| | | | |____.DS_Store
| | | | |____etc
| | | | | |____.DS_Store
```

**Comment by imaress** [comment]
Yep, deleting .DS_Store files like says @textbook fix the problem, but by default atom must ignore this files for squash directories if the hideIgnoredNames feature is true.
